packages/perseus/src/widgets/__tests__/numeric-input.test.ts

```diff
@@ -485,20 +485,6 @@ describe("Numeric input widget", () => {
         );
     });
 
-    it("calls the interaction callback on focus", () => {
```

Comment:
I think if we're going to remove this test, we should add a test for `numeric-input` in the new tests (which test `input-number` but not `numeric-input`).